voxel51/cli/cli.py

```diff
@@ -948,13 +1037,13 @@ def setup(parser):
             "-l", "--limit", metavar="LIMIT", type=int, default=-1,
             help="limit the number of analytics listed")
         parser.add_argument(
-            "-s", "--search", metavar="[FIELD:]STR",
+            "-s", "--search", metavar="SEARCH",
             help="search to limit results when listing analytics")
         parser.add_argument(
             "--sort-by", metavar="FIELD",
             help="field to sort by when listing analytics")
         parser.add_argument(
-            "-a", "--all-versions", action="store_true",
```

Comment:
I think `-a` is conventional enough as an "all" argument that keeping it around is fine

Reply:
okay I added it back.

I had removed it because both `--ascending` and `--all-versions` start with `a`, so it can be hard to remember which one has the `-a` shorthand